provisioning/roles/mailcatcher/meta/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+dependencies:
+  - { role: ruby }
+  - { role: supervisor, service_name: "mailcatcher", user: "root", command: "bash -c \"mailcatcher --http-ip 0.0.0.0 -f\"", root_directory: "/usr/local/bin/" }
```

Comment:
You can use single quotes `'` inside the command instead of escaping the double quotes.
Also I would use the absolute path of the `mailcatcher` binary